### DISCUSSION TITLE
I have changed the date format to day/monthname/year with Imask in angular but even with the custom logic in format and parse functions, It isn't allowing me tokey in full date

### DISCUSSION THREAD

**Discussion by sonali395** [uNmAnNeR/imaskjs#693] [Q&A]
This issue mentions the details https://github.com/uNmAnNeR/imaskjs/issues/692

**Comment by ivan-palatov** [comment]
For anyone who would stumble upon this, I would save u a bit of time.

Just pass in `lazy: false`. And it would solve the problem. Also, this is not specific to angular.
